app/assets/stylesheets/_tag_group.scss

```diff
@@ -0,0 +1,17 @@
+.tag-groups {
+  margin: 20px 0;
+
+  .color_container {
+    padding: 2px;
+    background: rgb(255, 255, 255);
+    border-radius: 1px;
+    box-shadow: rgba(0, 0, 0, 0.0980392) 0px 0px 0px 1px;
```

Comment:
What do you think about `$color-container-shadow-color`? It'll be easier to search for it later.